### ISSUE TITLE
Cannot read property `accept-version` when changing column from array to non-array

### ISSUE THREAD

**Issue by soedirgo** [supabase/postgres-meta#144]
- [x] Repro

**Comment by joshenlim** [comment]
This is a sample payload that I tried with, which returned the error

```
name: "test_column"
comment: "this is a comment"
type: "int4[]"
defaultValue: "{1, 2, 3}"
defaultValueFormat: "expression"
isIdentity: false
isNullable: true
```

![image](https://user-images.githubusercontent.com/19742402/131888940-2c2d5147-8152-4302-ae00-501fbc716ef2.png)

**Comment by joshenlim** [comment]
btw bump on this ^ i've a feeling its that pg-meta isn't returning the errors from postgres correctly? e.g if i have a column that has entries with null values in it, and I toggle is_nullable to false, this error gets thrown as well

**Comment by joshenlim** [comment]
Relevant issue: https://github.com/supabase/supabase/issues/592#issuecomment-769977569

**Comment by juanzgc** [comment]
I also reproduced this bug in the following steps:

1. Create a new table: 

![image](https://user-images.githubusercontent.com/24414523/133143409-fd44a881-77b0-42a2-a9bb-45eb127f661b.png)'

2. Save the new table.

3. Edit the `object` column and add the following Default value:

```
{'test':null}
```

![image](https://user-images.githubusercontent.com/24414523/133145925-855211d2-6194-4cd7-8928-88e1c33032d6.png)

4. Save the updated column and you will see the error.

![image](https://user-images.githubusercontent.com/24414523/133145996-27aa91f2-368f-4ae7-b2f1-f1866294ff32.png)

**Comment by soedirgo** [comment]
> i've a feeling its that pg-meta isn't returning the errors from postgres correctly?

This hits the mark, pg-meta wasn't returning the *actual* errors, which will be fixed in #149.

As for most of the errors when changing column types, the actual error seems to be that the types can't be converted automatically. E.g. changing a `text` column to `text[]` will give `column "my_column" cannot be cast automatically to type text[]`. The solution in this case is to drop down to SQL:
```sql
ALTER TABLE t ALTER my_column TYPE text[] USING ARRAY[my_column];
```

I think it's acceptable for the table editor to just error out when converting incompatible types, but maybe we should document how to get around this (e.g. by pointing them here).

**Comment by joshenlim** [comment]
gotcha on this, thanks so much @soedirgo! 🙏 Agreed that table editor should just return the errors from postgres rather than trying to do some magic to workaround the errors
